packages/teams-js/src/public/pages.ts

```diff
@@ -19,7 +19,7 @@ import { runtime } from './runtime';
  */
 export namespace pages {
   /**
-   * Return focus to the host. Will move focus forward or backward based on where the app container falls in
+   * Return focus to the host. Will move focus forward or backward based on where the application container falls in
    * the F6/Tab accessiblity loop in the host.
```

Comment:
I've never heard the term "Tab accessibility loop". [Seems like](https://accessibility.digital.gov/ux/tab-order/) the standard term is *tab order*. How about: "the F6 / tab order"